pbft/pbft.cpp

```diff
@@ -1122,7 +1122,7 @@ pbft::is_valid_viewchange_message(const pbft_msg& viewchange_message, const bzn_
     }
 
     auto valid_checkpoint_hashes = this->validate_and_extract_checkpoint_hashes(viewchange_message);
-    if (valid_checkpoint_hashes.empty() && viewchange_message.sequence() != 0)
+    if (valid_checkpoint_hashes.empty() || viewchange_message.sequence() == 0)
```

Comment:
This check means that any viewchange where there hasn't been a checkpoint yet will fail.

Reply:
I've re done the fix.